generate_isoscape.ipynb

```diff
@@ -11,7 +11,7 @@
     },
```

Comment:
<p>What happened to <code>required_geotiffs</code>? I see that it is specified in <code>training_column_names</code>--  is this going to be reflected in a future change to ddf-isoscapes? Is it already in and I just missed it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf_common/pull/50/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
SGTM